src/gpu/intel/jit/ir/tensor.hpp

```diff
@@ -535,10 +526,6 @@ class layout_t {
         return oss.str();
     }
 
-    IR_DEFINE_DUMP()
```

Comment:
What is the new way to dump a layout in GDB?

Reply:
In my common usage, I will just run
```
print layout.str()
```
We can definitely add back in the `dump()` interface later though. The main issue for this removal is to avoid the `IR_DEFINE_DUMP` macro in stable DSL interface.

Reply:
@rjoursler I think the main usage for `dump()` functionality was multi-line output which doesn't look nice with `print` in gdb. This includes `stmt_t`, `tensor_t`, etc. As a quick fix I would just migrate the macro (`DSL_DEFINE_DUMP`) - it doesn't seem to make the interface less stable, IMO.